javascript/ql/src/semmle/javascript/GlobalAccessPaths.qll

```diff
@@ -115,6 +116,11 @@ module GlobalAccessPath {
       // Note: Avoid unneeded materialization of DataFlow::Node.getFile()
       rhs.getAstNode().getFile() = file
     )
+    or
+    // Hard-code JSON methods from the externs file, since they aren't explicitly assigned.
```

Comment:
Wouldn't it be easier to just update `es5.js`? We control it.

Reply:
You're right let's do that. I'll do another eval to ensure nothing bad happens.